packages/models-library/src/models_library/utils/specs_substitution.py

```diff
@@ -14,6 +14,14 @@
 SubstitutionValue: TypeAlias = StrictBool | StrictInt | StrictFloat | str
 
 
+def _serializer(data: dict[str, Any]) -> str:
```

Comment:
MINOR: i recommend rename them as `_json_dumps` and `_json_loads` which suggest that are alternative of `json dumps/loads`